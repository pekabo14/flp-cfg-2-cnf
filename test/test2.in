E,T,F
a,b,c,d,i
E
E->EaT
E->T
T->TbF
T->F
F->cEd
F->i
