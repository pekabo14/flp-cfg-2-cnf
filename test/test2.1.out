E,T,F
a,b,c,d,i
E
E->EaT
E->TbF
E->cEd
E->i
T->TbF
T->cEd
T->i
F->cEd
F->i